src/command/help/help.c

```diff
@@ -224,6 +224,42 @@ helpRenderValue(const ConfigOption optionId, const unsigned int optionIdx)
     FUNCTION_TEST_RETURN(STRING, result);
 }
 
+static String *
+helpRenderValue(const ConfigOption optionId)
+{
+    FUNCTION_TEST_BEGIN();
+        FUNCTION_TEST_PARAM(ENUM, optionId);
+    FUNCTION_TEST_END();
+
+    String *result = helpRenderValueIdx(optionId, 0);
+
+    if (cfgOptionGroup(optionId))
+    {
+        MEM_CONTEXT_TEMP_BEGIN()
+        {
+            for (unsigned int optionIdx = 1; optionIdx < cfgOptionGroupIdxTotal(cfgOptionGroupId(optionId)); optionIdx++)
+            {
+                const String *const value = helpRenderValueIdx(optionId, optionIdx);
+
+                if (!strEq(result, value))
+                {
+                    MEM_CONTEXT_PRIOR_BEGIN()
+                    {
+                        strFree(result);
+                        result = strNewZ("<multi>");
```

Comment:
So this is the ultimate output if the values are different?  I guess I'd assumed it'd show all of the values in this case, but if we're able to query individually it's probably fine.

Reply:
(I see later that this is a sentinel value, so disregard.)